webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -84,14 +84,17 @@ export const LinkTooltip = ({href, connected}) => {
     if (data) {
         let date = new Date(data.created_at);
         date = date.toDateString();
-
         return (
             <div className='github-tooltip'>
-                <div className='github-tooltip box github-tooltip--large github-tooltip--bottom-left p-4'>
+                <div
+                    className='github-tooltip box github-tooltip--large github-tooltip--bottom-left p-4'
+                    style={{backgroundColor: theme.centerChannelBg, border: `1px solid ${hexToRGB(theme.centerChannelColor, '0.16')}`}}
```

Comment:
(0/5) Popover border uses --center-channel-color-20. Any reason to go for 16 instead of 20? Also, does the variable centerChannelColor20 (or anything similar) exist on the theme object?